src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioErrorReportingService.cs

```diff
@@ -31,43 +30,72 @@ internal partial class VisualStudioErrorReportingService : IErrorReportingServic
             _listener = listener;
         }
 
-        public void ShowErrorInfo(string message, params ErrorReportingUI[] items)
+        public void ShowErrorInfoInActiveView(string message, params ErrorReportingUI[] items)
```

Comment:
well, I can open issue for IDE to decide how you guys want to tweak the behavior. but it is not related to this fix.

and I actually like code fix message being show up in the text view that I invoked the code fix. I dont want those to pile up in the global/VS main info bar place.